tests/enums/src/main/graphql/java/schema.graphqls

```diff
@@ -0,0 +1,56 @@
+type Query {
+  direction: Direction
+  gravity: Gravity
+  foo: Foo
+  fooClass: FooClass
+  fooEnum: FooEnum
+}
+
+enum Direction {
+  NORTH,
+  north @deprecated,
+  SOUTH,
+  EAST,
+  WEST,
+
+  # renamed in extra.graphqls
+  type,
+
+  # Value names should be escaped with _ suffixes when generating enums
+  name,
+  ordinal,
+}
+
+enum Gravity {
+  TOP,
+  # fails with java.lang.AssertionError: Class with incorrect id found: expected enums/type/Gravity.TOP, actual enums/type/Gravity.top
+  # See https://youtrack.jetbrains.com/issue/KT-49352
+  # top @deprecated,
+  top2 @deprecated,
+  BOTTOM,
+  LEFT,
+  RIGHT,
+
+  # Value names should not be escaped when generating sealed classes
+  name,
+  ordinal,
+
+  # renamed in extra.graphqls
+  type,
+}
+
+# See https://youtrack.jetbrains.com/issue/KT-52315
+enum Foo {
+  header,
+  footer,
+}
+
+enum FooEnum {
+  # not renamed in extra.graphqls, will be renamed automatically
+  type,
+}
+
+enum FooClass {
+  # not renamed in extra.graphqls, will be renamed automatically
+  type,
+}
```

Comment:
Do you need to duplicate the schema? Looks like it's the same as the Kotlin one?

Reply:
No, you're right! Updated.